File: app/src/main/java/com/example/bankaccount/Account.java
package com.example.bankaccount;

public class Account {
    private int id;
    private String accountName;
    private String amount;
    private String iban;
    private String currency;

    public Account() {
    }

    public Account(int id, String accountName, String amount, String iban, String currency) {
        this.id = id;
        this.accountName = accountName;
        this.amount = amount;
        this.iban = iban;
        this.currency = currency;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getAccountName() {
        return accountName;
    }

    public void setAccountName(String accountName) {
        this.accountName = accountName;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getIban() {
        return iban;
    }

    public void setIban(String iban) {
        this.iban = iban;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }
}

File: app/src/main/java/com/example/bankaccount/Account.kt
package com.example.bankaccount

class Account {
    var id = 0
    var accountName: String? = null
    var amount: String? = null
    var iban: String? = null
    var currency: String? = null

    constructor() {}
    constructor(id: Int, accountName: String?, amount: String?, iban: String?, currency: String?) {
        this.id = id
        this.accountName = accountName
        this.amount = amount
        this.iban = iban
        this.currency = currency
    }
}